### PR TITLE
security.txt #882

### LINKED ISSUE #882: create security.txt
https://securitytxt.org/

### PR DESCRIPTION
**Pull request by guaka**
#### Proposed Changes

* security.txt

#### Testing Instructions

*

Fixes #882

### CODE REVIEW

**Comment by simison** [comment]
There's already https://github.com/Trustroots/trustroots/blob/e40663d48338e8fc2d4596462b0dfd44f3a60028/SECURITY.md

**Comment by guaka** [comment]
`security.txt` seems to be picking up. And it doesn't hurt in any way.
See also https://twitter.com/search?q=security.txt&src=typd

File: public/.well-known/security.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# If you would like to report a security issue
+# you may report it to us by email
+Contact: hello@trustroots.org
+Hiring: https://www.trustroots.org/volunteering
+
+
+# See also https://github.com/Trustroots/trustroots/blob/e40663d48338e8fc2d4596462b0dfd44f3a60028/SECURITY.md
```
Comment (simison) [inline public/.well-known/security.txt]: Let's add non-hash URL to link to the latest version:
https://github.com/Trustroots/trustroots/blob/master/SECURITY.md

**Comment by simison** [comment]
> security.txt seems to be picking up. And it doesn't hurt in any way.
> See also https://twitter.com/search?q=security.txt&src=typd

Sure, makes sense. 👍

**Pull request merged**